runtime/doc/test-coverage.md

```diff
@@ -134,6 +134,32 @@ Check whether the digital signature returned by Sign command can be verified usi
 Check that the Sign command fails on simulated contexts as simulation context do not allow signing | **TestSignSimulation** | StatusInvalidArgument
 Calls and tests behavior of the DPE command Sign with the Symmetric flag set | **TestSignSymmetric** | N/A
 
+<br><br>
```

Comment:
I think you might be using an old version of this file, since it no longer has a mailbox command tests or wycheproof tests section. Can you add these tests to the pcr tests section?